### ISSUE TITLE
How to run YOLO with onnxruntime

### ISSUE THREAD

**Issue by freedom9393** [microsoft/onnxruntime#6236]
How to run YOLO with onnxruntime? Is it possible? Is there any example of doing it? I'm actually trying to deply YOLO model to Intel (integrated) GPU, so I'm trying onnxruntime to achieve that. Any seggestions?

**Comment by snnn** [comment]
Yes we support yolov2, v3 and v4. You can find more details at: https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/yolov4

They have example code for  preprocessing and postprocessing.   You need to connect them with https://www.onnxruntime.ai/python/tutorial.html

**Comment by freedom9393** [comment]
@snnn, Thanks a lot dude. So, I can deploy this to Intel (integrated) GPU? Or could you suggest me anything for that? Thanks in advance.

**Comment by KyloEntro** [comment]
Is there a sample written in C++ to post process yolo detection result ?

**Comment by freedom9393** [comment]
@KyloEntro I tried another way, take a look [this link](https://github.com/TNTWEN/OpenVINO-YOLOV4). I'm sorry for shared a link that is not related to onnxruntime, I'm just trying to help someone who needs help
